File: message-api-schema-validator/src/main/java/uk/ac/jisc/rdss/tests/vocabulary/read/VocabularyReadResponseTest.java
package uk.ac.jisc.rdss.tests.vocabulary.read;

import org.junit.Test;

import uk.ac.jisc.rdss.tests.AbstractSchemaValidatorTest;

public class VocabularyReadResponseTest extends AbstractSchemaValidatorTest {

    @Override
    protected String getSchemaFileName() {
        return "messages/vocabulary/read/response_schema.json";
    }

    @Test
    public void validateResponse() throws Exception {
        validateJson("messages/vocabulary/read/response.json");
    }
}

File: message-api-schema-validator/tests/vocabulary/vocabulary_read_response_test.py
from tests.abstract_schema_validator_test import AbstractSchemaValidatorTest
from unittest import TestCase


class VocabularyReadResponseTest(AbstractSchemaValidatorTest, TestCase):
    def get_json_schema_file_name(self):
        return 'messages/vocabulary/read/response_schema.json'

    def runTest(self):
        self.validate_json('messages/vocabulary/read/response.json')
